app/controllers/shelves_controller.rb

```diff
@@ -103,4 +103,20 @@ def dissociate
       end
     end
   end
+
+  def check_trays_new
+    @shelf = Shelf.new
+  end
+
+  def check_trays
+    begin
+      @shelf = GetShelfFromBarcode.call(params[:shelf][:barcode])
+      @scanned = []
+    rescue StandardError => e
+      notify_airbrake(e)
```

Comment:
We're no longer using airbrake. Use the following for most exception handling:

```suggestion
      Raven.capture_exception(e)
```

There are other examples in the app of how we can use Raven to log errors. Take a look at the other examples and there is online documentation at Sentry.io